backend/src/routes/aggregators/applicationsOCP.ts

```diff
@@ -0,0 +1,163 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { getPagedSearchResources } from '../../lib/search'
+import { IResource } from '../../resources/resource'
+import { getKubeResources } from '../events'
+import { ApplicationCacheType, generateTransforms } from './applications'
+
+const query = {
+  operationName: 'searchResult',
+  variables: {
+    input: [
+      {
+        filters: [
+          {
+            property: 'kind',
+            values: ['Deployment'],
+          },
+        ],
+        limit: 200000,
```

Comment:
I think this makes sense because for Argo we are getting the app definition directly so less resources to pull versus OpenShift apps where we need to gather as much resources as possible and calculate the apps from those.

Reply:
externalized limit